elife/config/etc-init-goaws-init.conf

```diff
@@ -0,0 +1,11 @@
+description "GOAWS will run a local AWS server that can be used by setting --endpoint=http://localhost:4100 to commands"
+respawn
+kill timeout 10
+start on (local-filesystems and net-device-up IFACE!=lo)
+setuid {{ pillar.elife.deploy_user.username }}
+env HOME=/home/{{ pillar.elife.deploy_user.username }}
+chdir /home/{{ pillar.elife.deploy_user.username }}
+script
+    exec mkdir -p -m u=rwx /var/run/goaws
```

Comment:
`exec` replaces the current process with the one being executed, therefore it should only be used for the last command in the `script` stanza. The content of `script...end script` is just a shell script, so `mkdir ...` is fine.